.gitignore

```diff
@@ -204,3 +204,4 @@ src/TraceEvent/EventCounterHandler.cs
 OSExtensions.cs
 password.txt
 documentation/internal/
+*.userosscache
```

Comment:
Is this intentional?  I've not seen this type file show up anywhere.

Reply:
@ninedan, can you comment on the gitignore change?  Thanks.

Reply:
Yes, the .userosscache file seems to be generated by Visual Studio.

Reply:
Thanks much.